scripts/run_integration_tests_aarch64.sh

```diff
@@ -192,6 +192,15 @@ update_workloads() {
     fi
     popd
 
+    # Download Cloud Hypervisor binary from its last stable release
+    LAST_RELEASE_VERSION="v23.0"
+    CH_RELEASE_URL="https://github.com/cloud-hypervisor/cloud-hypervisor/releases/download/$LAST_RELEASE_VERSION/cloud-hypervisor-static-aarch64"
+    CH_RELEASE_NAME="cloud-hypervisor-static-aarch64"
+    pushd $WORKLOADS_DIR
+    time wget --quiet $CH_RELEASE_URL || exit 1
```

Comment:
I think I find something wrong here.

You know AArch64 use bare-metal for the integration test. The file with the same name could exist. `wget` doesn't replace the old file, but renames the new file to `cloud-hypervisor-static-aarch64.X` in this case.

Can you change it and try again?

`time wget --quiet $CH_RELEASE_URL -O "$CH_RELEASE_NAME" || exit 1`